package.json

```diff
@@ -63,23 +66,25 @@
         "uuid": "^3.3.2"
     },
     "devDependencies": {
-        "@nestjs/testing": "^6.0.0",
+        "@libero/eslint-config": "^0.0.1",
         "@types/dotenv": "^6.1.1",
-        "@types/express": "^4.16.0",
+        "@types/express": "^4.17.2",
+        "@types/helmet": "^0.0.45",
         "@types/jest": "^23.3.13",
+        "@types/jsonwebtoken": "^8.3.7",
+        "@types/knex": "^0.16.1",
         "@types/lodash": "^4.14.149",
         "@types/node": "^10.12.18",
         "@types/node-fetch": "^2.5.4",
         "@types/supertest": "^2.0.7",
```

Comment:
Wondering if its worth auditing the dependencies to see if any are unused now? Packages like `sqlite3` might be removable